src/request.ts

```diff
@@ -177,6 +179,11 @@ class Request {
               options.headers.authorization = `Bearer ${accessToken}`;
             }
           }
+
+          const proxyUrl = Cli.getInstance().config.get(settingsNames.proxyUrl);
```

Comment:
Shall we add here fallback based on standard Node.js env vars like mentioned in https://github.com/pnp/cli-microsoft365/issues/2698#issuecomment-1486674217?

Reply:
@pnp/cli-for-microsoft-365-maintainers, anyone happen to know more about this?

Reply:
Setting environment variable differs per shell and I don't think we can do anything about it. I'd count it towards the knowledge that users need to work with the shell of their choice, that goes beyond CLI, and nothing that we should handle.